d2common/Configuration.go

```diff
@@ -2,54 +2,129 @@ package d2common
 
 import (
 	"encoding/json"
-	"io/ioutil"
-	"log"
+	"fmt"
 	"os"
-	"path"
-	"strings"
+	"path/filepath"
+	"runtime"
 
 	"github.com/mitchellh/go-homedir"
 )
 
-// Configuration defines the configuration for the engine, loaded from config.json
+// Configuration defines the configuration for the engine
 type Configuration struct {
-	Language        string
-	FullScreen      bool
-	Scale           float64
-	RunInBackground bool
-	TicksPerSecond  int
-	FpsCap          int
-	VsyncEnabled    bool
-	MpqPath         string
-	MpqLoadOrder    []string
-	SfxVolume       float64
-	BgmVolume       float64
+	Language          string
+	FullScreen        bool
+	Scale             float64
+	PauseInBackground bool
+	TicksPerSecond    int
+	FpsCap            int
+	VsyncDisabled     bool
```

Comment:
I had to switch these so that it would work with Go default values.

Reply:
That's good. Making the zero-value the default value.